### ISSUE TITLE
Non-UTF8 strings not supported

### ISSUE THREAD

**Issue by SpaceManiac** [SpaceManiac/SpacemanDMM#7]
```
code\modules\clothing\suits\vg_suits.dm, line 71, column 32:
non-utf8 string
```

https://github.com/Citadel-Station-13/Citadel-Station-13/blob/89fa4b0f28ac62c07ca85a02c4110d49bbbb0c0d/code/modules/clothing/suits/vg_suits.dm#L71

Needs to not try to decode strings and just keep bytebuffers instead

**Comment by SpaceManiac** [comment]
Hack fix in e943a0b, real fix still needed

**Comment by SpaceManiac** [comment]
As of 5857a2e, string literals are interpreted as Latin-1.
